### ISSUE TITLE
Minecraft Jar not found: falling back on placeholder textures.

### ISSUE THREAD

**Issue by d3dx9** [leMaik/ChunkyMap#12]
Hi,

I'm trying to run ChunkyMap on PaperMC 1.16.1 and getting some errors dropped.
```

[23:50:35 WARN]: [ChunkyMap] Rendering tile failed
java.util.concurrent.ExecutionException: de.lemaik.chunkymap.rendering.RenderException: Could not get final image from Chunky
        at java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:395) ~[?:?]
        at java.util.concurrent.CompletableFuture.get(CompletableFuture.java:1999) ~[?:?]
        at de.lemaik.chunkymap.dynmap.ChunkyMapTile.render(ChunkyMapTile.java:78) ~[?:?]
        at org.dynmap.MapManager$FullWorldRenderState.processTile(MapManager.java:804) ~[?:?]
        at org.dynmap.MapManager$FullWorldRenderState.access$1300(MapManager.java:262) ~[?:?]
        at org.dynmap.MapManager$FullWorldRenderState$3.call(MapManager.java:686) ~[?:?]
        at org.dynmap.MapManager$FullWorldRenderState$3.call(MapManager.java:684) ~[?:?]
        at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304) ~[?:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
        at java.lang.Thread.run(Thread.java:834) [?:?]
Caused by: de.lemaik.chunkymap.rendering.RenderException: Could not get final image from Chunky
        at de.lemaik.chunkymap.rendering.local.ChunkyRenderer.lambda$render$0(ChunkyRenderer.java:73) ~[?:?]
        at se.llbit.chunky.renderer.RenderManager.pathTraceLoop(RenderManager.java:228) ~[?:?]
        at se.llbit.chunky.renderer.RenderManager.run(RenderManager.java:162) ~[?:?]
Caused by: java.lang.NoSuchMethodException: se.llbit.chunky.renderer.scene.Scene.computeAlpha(se.llbit.util.TaskTracker)
        at java.lang.Class.getDeclaredMethod(Class.java:2475) ~[?:?]
        at de.lemaik.chunkymap.rendering.local.ChunkyRenderer.getImage(ChunkyRenderer.java:93) ~[?:?]
        at de.lemaik.chunkymap.rendering.local.ChunkyRenderer.lambda$render$0(ChunkyRenderer.java:71) ~[?:?]
        at se.llbit.chunky.renderer.RenderManager.pathTraceLoop(RenderManager.java:228) ~[?:?]
        at se.llbit.chunky.renderer.RenderManager.run(RenderManager.java:162) ~[?:?]
```

Also it cant download the minecraft.jar for the tiles.

May you can help?

**Comment by leMaik** [comment]
@d3dx9 Thanks for reporting this (didn't see this issue until now). It'll be fixed by #13 and I'll release a fixed version when it is merged.

**Comment by leMaik** [comment]
Fixed in [v2.4.0](https://github.com/leMaik/ChunkyMap/releases/tag/v2.4.0)
